src/main/java/taxi/service/DriverService.java

```diff
@@ -1,6 +1,8 @@
 package taxi.service;
 
+import java.util.Optional;
 import taxi.model.Driver;
 
 public interface DriverService extends GenericService<Driver> {
+    Optional<Driver> findByLogin(String login);
```

Comment:
```suggestion
    Driver findByLogin(String login);
```
Don't use Optional in services